src/views/virtualmachines/details/tabs/configuration/network/NetworkTab.tsx

```diff
@@ -37,22 +38,20 @@ const NetworkInterfaceListPage: FC<NetworkInterfaceListPageProps> = ({ obj: vm }
   );
 
   return (
-    <div className="network-interface-list-page">
-      <ListPageBody>
-        <SidebarEditor
-          onResourceUpdate={onSubmit}
-          pathsToHighlight={PATHS_TO_HIGHLIGHT.NETWORK_TAB}
-          resource={vm}
-        >
-          <Title className="network-interface-list-page__title" headingLevel="h2">
-            {t('Network interfaces')}
-          </Title>
-          <AddNetworkInterfaceButton vm={vm} />
-          <NetworkInterfaceList vm={vm} />
-        </SidebarEditor>
-      </ListPageBody>
-    </div>
+    <>
+      <SidebarEditor
+        onResourceUpdate={onSubmit}
+        pathsToHighlight={PATHS_TO_HIGHLIGHT.NETWORK_TAB}
+        resource={vm}
+      >
+        <PageSection>
+          <Title headingLevel="h2">{t('Network')}</Title>
```

Comment:
I'd use `{t('Network interfaces')}` here, as originally. There is no any reason to use only 'Network'. You still have used the original longer title for "Scheduling" tab, so why not also here :)